bluesky/run_engine.py

```diff
@@ -508,7 +508,13 @@ def __call__(self, plan, subs=None, *, raise_if_interrupted=False,
         if not self.state.is_idle:
             raise RuntimeError("The RunEngine is in a %s state" % self.state)
 
-        futs = [f for sup in self.suspenders for f in sup.get_futures()]
+        futs = []
+        for sup in self.suspenders:
+            f_lst, msg = sup.get_futures()
+            if f_lst:
+                futs.extend(f_lst)
+                print(msg)
```

Comment:
Correct.  We already get _any_ message when that trips.  The we were getting _nothing_ (just an apparent hang) when starting the scan.